src/modules/core/components/ColorSelect/ColorSelect.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useCallback, useMemo, ReactNode, ComponentProps } from 'react';
+import { defineMessages } from 'react-intl';
+
+import { Select, SelectOption, Form } from '~core/Fields';
+import ColorTag, { Color } from '~core/ColorTag';
+
+import styles from './ColorSelect.css';
+
+const MSG = defineMessages({
+  labelColorSelect: {
+    id: 'ColorSelect.labelColorSelect',
+    defaultMessage: 'Select color',
+  },
+});
+
+interface FormValues {
+  activeColor: Color;
+}
+
+interface Props {
+  /** Should `select` be disabled */
+  disabled?: boolean;
+
+  /** Active color */
+  activeOption: Color;
+
+  /** Callback function, called after value is changed */
+  onColorChange?: (color: Color) => any;
+}
+
+const displayName = 'ColorSelect';
+
+const ColorSelect = ({ disabled, activeOption, onColorChange }: Props) => {
+  const onChange = useCallback(
+    (color: Color) => {
+      if (onColorChange) {
+        return onColorChange(color);
+      }
+      return null;
+    },
+    [onColorChange],
+  );
+
+  const renderActiveOption = useCallback<
+    (option: SelectOption | undefined, label: string) => ReactNode
+  >(() => {
+    return <ColorTag color={activeOption || Color.Yellow} />;
+  }, [activeOption]);
+
+  const options = useMemo<ComponentProps<typeof Select>['options']>(() => {
+    const colors = Object.keys(Color).filter(
+      (val) => typeof Color[val as any] === 'number',
+    );
+    return [
+      ...colors.map((color) => {
+        return {
+          children: <ColorTag color={Color[color]} />,
+          label: `${color}`,
+          value: `${Color[color]}`,
+        };
+      }),
+    ];
+  }, []);
+
+  return (
+    <Form<FormValues>
+      initialValues={{
+        activeColor: activeOption || Color.Yellow,
+      }}
+      onSubmit={() => {}}
+    >
+      <div className={styles.wrapper}>
+        <Select
```

Comment:
There's a bug where after either closing (by clicking outside) or closing by selecting a new color, the select component retains focus:

![bug-color-select-focust-state](https://user-images.githubusercontent.com/1193222/101671533-4772af80-3a5d-11eb-9974-1b705d6f571b.gif)

Notice how I need to click again, outside of the select component in order to get rid of the focus.

The focus on the element should be lost when closing it.

Reply:
I don't think its a bug. It current implementation of Select.ts component:

  const close = useCallback(() => {
    setIsOpen(false);
    setSelectedOption(-1);
    // if (comboboxRef.current) {
    //   comboboxRef.current.focus();
    // }
  }, []);
I can remove if and it will fix problem - if we want